deploy/operator.yaml

```diff
@@ -29,3 +29,5 @@ spec:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
               value: "devopsconsole-operator"
+            - name: APP_SERVICE_IMAGE_NAME
+              value: "jarifibrahim/boilerplate-app" # change this to actual image when we have it
```

Comment:
This has to be changed to actual image when we have it

Reply:
Do we have a repo in quay? Then I can create one. As for the redhat-developer/app-service, it is already built for every new master revision on docker hub: https://cloud.docker.com/u/konradkleine/repository/docker/konradkleine/app-service

Reply:
I've opened this PR to create the repos on Quay https://gitlab.cee.redhat.com/service/app-interface/merge_requests/238/diffs#71dd9efa997ab16894984e6781c01dc898af5e13_0_33

Reply:
> https://gitlab.cee.redhat.com/service/app-interface/merge_requests/238/diffs#71dd9efa997ab16894984e6781c01dc898af5e13_0_33

This PR was closed a few days ago. Are we publishing the image somewhere?

Reply:
Updated the image.